package.json

```diff
@@ -24,13 +24,13 @@
   "author": "无惟",
   "license": "MIT",
   "dependencies": {
-    "butterfly-dag": "~4.1.0"
+    "butterfly-dag": "~4.1.22"
   },
   "peerDependencies": {
-    "react": ">15.6.1",
-    "react-dom": ">15.6.1",
-    "lodash": "^4.17.20",
-    "jquery": ">3.5.1"
+    "react": "~17.0.2",
```

Comment:
You can not change here.Many user use react15 & 16 . here support react 17.x

Reply:
ok. well this was the idea of the PR to go to latest react version. then reject it.